### PR TITLE
Kz globals fixes

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: core/kazoo_globals/src/kz_global.erl [hunk 1]
```diff
@@ -115,36 +126,43 @@ all_globals_by_node(Table, Node) ->
...
+node(#kz_global{node=Node}) -> Node.
 
-timestamp(#kz_global{timestamp=Timestamp}) ->
-    Timestamp.
+-spec timestamp(global()) -> integer().
+timestamp(#kz_global{timestamp=Timestamp}) -> Timestamp.
 
+-spec new_timestamp() -> integer().
+new_timestamp() ->
+    erlang:system_time('micro_seconds').
```
Comment (fenollp) [inline core/kazoo_globals/src/kz_global.erl]: new_timestamp is exactly kz_util:now_us/0

**Comment by jamesaimonetti** [comment]
@fenollp updated.

**Comment by fenollp** [comment]
Ah didn't notice now_us was /1. Can you make a follow-up PR replacing kz_util:now_*s/1 with the /0 counterparts?

**Comment by jamesaimonetti** [comment]
@fenollp see #2177

**Pull request merged**